community/slock/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh -e
+
+sed 's/usr\/local/usr/g' config.mk
```

Comment:
I take it you want to edit the file in-place? You can also use any character as a separator in `sed`. Saves having to escape the forward-slash.

```sh
sed -i'' 's|usr/local|usr|g' config.mk
```